docs/UserGuide.md

```diff
@@ -418,7 +418,9 @@ Examples:
 * `sort n`
 * `sort p`
 
-![result for 'sort'](images/sortResult.png)
+This screenshot shows the result of executing `sort p`.
```

Comment:
To standardise this, may need to check if other command section contains the explanation of screenshot as well.

Reply:
After offline discussion, we decide to standardise and add this line to all other commands!